### PR TITLE
Improve accessibility for lobid-gnd

### PR DESCRIPTION
**Pull request by fsteeg**
See https://github.com/hbz/lobid/issues/423

### CODE REVIEW

File: app/views/details.scala.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @import play.api.libs.json._
 
 @image() = {
-    <img src='https://lobid.org/imagesproxy?url=@resource.getImage.image' style="max-height:500px;" class="center-block"/>
+    <img src='https://lobid.org/imagesproxy?url=@resource.getImage.image' style="max-height:500px;" class="center-block" alt="resource.preferredName"/>
     <div class='text-center small'><br/>@Html(resource.getImage.label)</div>
```
Comment (dr0i) [inline app/views/details.scala.html:9]: Is it possible to actually get "resource.preferredName"? Prefixing this with "Abbildung von " would provide the user with meaningful data.

Comment (dr0i) [reply app/views/details.scala.html:9]: I think it's possible and you just forgot the `@`, right?

Comment (fsteeg) [reply app/views/details.scala.html:9]: Right, thanks! Fixed in https://github.com/hbz/lobid-gnd/pull/271/commits/82eb1955b8a0c79ad9ac33d7ade9b4851d99b52b.

**Comment by dr0i** [comment]
+1

**Pull request merged**